src/notifications/screens/notifications.screen.js

```diff
@@ -187,6 +187,7 @@ class Notifications extends Component {
     markAsReadByDispatch(notification.id);
     navigation.navigate('Issue', {
       issueURL: notification.subject.url.replace('pulls', 'issues'),
+      isPR: notification.subject.type === 'PullRequest',
```

Comment:
@lex111 Yea that crash is a pain in the butt to debug. I was experiencing it for a while, and eventually I think I found out that it was because I had something like

```
{ foo && <ReactElement /> }
```

where `foo` was a string instead of a boolean. IIRC changing `foo` to `!!foo` worked, but I'm not positive.